README.md

```diff
@@ -310,6 +310,7 @@ Table of Contents
   * [Prismic](https://www.prismic.io/) — Headless CMS. Content management interface with fully hosted and scalable API. The Community Plan provides 1 user with unlimited API calls, documents, custom types, assets, and locales. Everything that you need for your next project. Bigger free plans available for Open Content/Open Source projects.
   * [sanity.io](https://www.sanity.io/) – Hosted backend for structured content with customizable MIT licensed editor built with React. Unlimited projects. 3 users, 2 datasets, 500k API CDN requests, 5GB assets for free per project
   * [sensenet](https://sensenet.com) - API-first headless CMS providing enterprise-grade solutions for businesses of all size. The Developer plan provides 3 users, 500 content items, 3 built-in roles, 25+5 content types, fully accessible REST API, document preview generation and Office Online editing.
+  * [Strapi](https://www.strapi.io.com/) — Strapi is the leading open-source headless CMS. It’s 100% Javascript, fully customizable and developer-first. The community plan is free forever.
```

Comment:
whats the SaaS angle here? Seems more like a self hosted solution to me

Reply:
it doesn't work for me.
https://www.strapi.io.com/

But https://www.strapi.io works 😃

Reply:
Oh my bad, I thought it was listing any kind of software...